State.lua

```diff
@@ -234,6 +243,13 @@ function WarpDeplete:ResetCurrentPull()
 	self:SetForcesPull(0)
 end
 
+function WarpDeplete:ResetForceCountBooleans()
```

Comment:
`Booleans` doesn't really explain what this function does. Maybe `ResetForceCountEvents` or `ResetForceCountTriggers` would be a better name?